scripts/issue-genesis-utxo-expenditure.sh

```diff
@@ -32,6 +32,10 @@ args=" --real-pbft
        --wallet-key          ${from_key}
        --rich-addr-from    \"${from_addr}\"
        --txout            (\"${addr}\",${lovelace})
+       --topology            configuration/simple-topology.json
```

Comment:
Why is topology necessary for `cardano-cli` now?

Reply:
See: https://github.com/input-output-hk/cardano-node/pull/289#discussion_r344181423